src/test/resources/translations.json

```diff
@@ -4,10 +4,9 @@
     "es":"",
     "el":"<p>Ζήτησε να καταθέσεις το εξής ποσό: {{amount}}€.</p>\n\n<p>Θα ενημερωθείς μόλις επεξεργαστούμε το αίτημα σου.</p>\n"
   },
-  "depositTemplate":{
-    "en":"{{#minted}}\n    <p>Your deposit has been approved and your funds are ready.</p>\n{{/minted}}\n{{^minted}}\n    <p>Your deposit request has been rejected. Try to create new deposit request or contact support for more information.</p>\n{{/minted}}",
-    "es":"{{#minted}}\n  <p>Su depósito ha sido autorizado y sus fondos están disponibles.</p>\n {{/minted}}\n {{^minted}}\n  <p>Su solicitud de depósito ha sido rechazada. Intente crear una nueva solicitud de depósito o contacte con soporte para más información.</p>\n {{/minted}}",
-    "el":"{{#minted}}\n    <p>Η κατάθεσή σου έχει εγκριθεί και το κεφάλαιό σου είναι έτοιμο.</p>\n{{/minted}}\n{{^minted}}\n    <p>Το αίτημά σου απορρίθφηκε. Κάνε ένα νέο αίτημα κατάθεσης ή επικοινώνησε μαζί μας για περισσότερες πληροφορίες.</p>\n{{/minted}}\n"
+  "depositTemplate": {

```

Comment:
Add to file on drive with translations.

Reply:
I added it, we are using this drive from now on: https://docs.google.com/spreadsheets/d/1cFOL4B0kQ1XxulbMQBjonlJa6xjbDi8keU6jahCt0Z4/edit